en/debug-kit.rst

```diff
@@ -282,3 +282,8 @@ Helper Functions
 
 * `sql()` Dumps out the SQL from an ORM query.
 * `sqld()` Dumps out the SQL from an ORM query, and exits.
+
+Know Problems
+=============
+
+If you work with xDebug enabled you'll end up with an exception and stack trace (https://github.com/cakephp/debug_kit/issues/388). This problem can be avoided with setting ``xdebug.collect_params`` to ``1``.
```

Comment:
technically there is also a comma after enabled.

Reply:
and "with setting" is "by setting"

Reply:
``` to ``0`` or ``1`` ```

0 is default, and is also OK, as stated in https://github.com/cakephp/debug_kit/issues/388#issuecomment-463139803